### DISCUSSION TITLE
Reinsert the straight, simple connector

### DISCUSSION THREAD

**Discussion by Arivit96** [jgraph/drawio#4039] [General]
[x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
[x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**BUG**
The simple, straight connection without any label not angles and curve was recently removed from the product. Straight arrow there isn't in library.

**VERSION**
- draw.io version 22.1.9

<img width="239" alt="Schermata 2023-12-14 alle 21 31 37" src="https://github.com/jgraph/drawio/assets/64424005/b031c790-1e83-42a0-ad84-72696aa9e20b">

**Comment by davidjgraph** [comment]
It's not a bug, why would you create an edge from the sidebar?

**Reply by Arivit96** [reply]
@davidjgraph do not compare the single arrow in the left sidebar

It's the same problem as topic #2304

Issue #2308

![IMG_0338](https://github.com/jgraph/drawio/assets/64424005/cfada09d-3163-46bf-a775-706c12c09001)

**Comment by alderg** [comment]
@Arivit96 Are you using a defaultEdgeStyle in your configuration (via Extras, Configuration)?
